src/schema.js

```diff
@@ -228,7 +228,7 @@ const likePostSchema = Joi.object({
 			.required()
 	}
 })
-const unlikePostSchema = Joi.object({
+const unlikeOrUnflagPostSchema = Joi.object({
```

Comment:
i actually skipped this part because I see why we're trying to not make it repetitive and it looked okay to me. 

However, for now, let's separate the concerns though they look the same. takes away the need to worry if one changes for some reason.

also personally I find the single naming more concise and makes it easier for me to navigate the code from the routes.